examples/seismic/acoustic/acoustic_example.py

```diff
@@ -2,34 +2,21 @@
 from argparse import ArgumentParser
 
 from devito.logger import info
-from devito import Constant
+from devito import Constant, Eq, Function, Operator
 from examples.seismic.acoustic import AcousticWaveSolver
 from examples.seismic import demo_model, TimeAxis, RickerSource, Receiver
 
 
-# Velocity models
-def smooth10(vel, shape):
+def smooth(dest, src):
```

Comment:
Basically its hard to find anything to like about this function - to be fair the original was also also far from ideal. Why has the radius of the smoothing kernel been dropped? Why are we not using a standard Gaussian smoothing filter (ie applying principle of least astonishment)?

Reply:
the maximum radius based on the amount of halo available is automatically picked by ``src.avg``. Otherwise you (user) can pick your own, as long as you know what you're doing. 

As you said, I simply translated the original function into the Devito language, and generalised it a little bit. Nothing more nothing less